src/Microsoft.ML.Featurizers/DateTimeTransformer.cs

```diff
@@ -643,13 +655,15 @@ private sealed class Mapper : MapperBase
             private readonly DateTimeTransformer _parent;
             private ConcurrentDictionary<long, TimePoint> _cache;
             private ConcurrentQueue<long> _oldestKeys;
+            private readonly DateTime _unixEpoc;
```

Comment:
This should be a `static readonly` field. No need to new one up every time.

Reply:
Changed it as suggested.